src/praisonai-sandbox/praisonai_sandbox/novita.py

```
@@ -0,0 +1,302 @@
+        if not self.is_available:
+            raise RuntimeError(
+                f"Novita is not available. Please install novita-sandbox and set NOVITA_API_KEY. {_INSTALL_HINT}"
+            )
+
+        try:
+            from novita_sandbox.core import AsyncSandbox
+        except ImportError:
+            raise ImportError(
+                "novita-sandbox not installed. "
+                "Install with: pip install novita-sandbox"
+            )
```

Ruff (0.16.1)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)